presto-product-tests/src/main/resources/sql-tests/testcases/window_functions/windowSpecificationAABcp.sql

```diff
@@ -0,0 +1,8 @@
+-- database: presto; groups: window;
```

Comment:
The test name `WindowSpecificationAABcp` does tell me nothing. Please add a comment above the query with explanation of test purpose.